common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/about.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
I think it better to use EPL 2.0 rather than 1.0

Reply:
License choice is done at project level. Other bundle are using EPL 1.0.